File: ProyectoFinal.Antares.Domain/Modelos/Empleado.cs
using ProyectoFinal.Antares.Domain.Enums;

namespace ProyectoFinal.Antares.Domain.Modelos;

public class Empleado : BaseModel
{
    public string Nombre { get; set; } = null!;

    public string Dni { get; set; } = null!;

    public bool Activo { get; set; }
    
    public TipoEmpleado Tipo { get; set; }
}

File: FrontendWeb/ProyectoFinal_Antares_FrontEnd_Web/src/app/Models/empleado.ts
export interface Empleado
{
    id: number;
    nombre: string;
    dni: string;
    activo: boolean;
    tipo: TipoEmpleado;
}

export enum TipoEmpleado{
    Administrador,
    Gerente,
    Mozo,
    Cajero,
    Barman
}
